app/views/pageflow/social_share/_entry_meta_tags.html.erb

```diff
@@ -2,10 +2,9 @@
 
 <%= social_share_entry_image_tags(entry) %>
 
-<%= tag :meta, :property => "og:title", :content => "#{entry.title} - #{entry.theming.cname_domain}" %>
+<%= tag :meta, :property => "og:title", :content => pretty_entry_title(entry) %>
 <%= tag :meta, :property => "og:description", :name => "description", :content => social_share_entry_description(entry) %>
 <%= tag :meta, :property => "og:url", :content => pretty_entry_url(entry) %>
-<%= tag :meta, :property => "og:site_name", :content => entry.theming.cname_domain %>
```

Comment:
Always scared to change a winning team ;) For what it's worth, also not really related to this change.